clean: Improve troubleshooting for missing organization CY-5678 (#1051)
codacy/docs: docs/faq/general/why-cant-i-see-my-organization.md

# Why can't I see my organization?

If you don't see your organization when [adding your organization on Codacy](../../organizations/what-are-synced-organizations.md#adding-an-organization), try the following steps:
Follow the steps below if you don't see your organization while [adding your organization on Codacy](../../organizations/what-are-synced-organizations.md#adding-an-organization):

1.  Make sure you have access to the organization with the account you're logged in.
1.  Make sure that you have access to the organization on the Git provider using the account that you used to log in on Codacy.

1.  **If you're using GitHub** [install and authorize Codacy on your organization](https://github.com/apps/codacy-production/installations/new).

> 1.  Refresh the list of organizations on Codacy by clicking **refresh this list** on the Organizations page:

Reviewer: I would keep this option separate from the other and appearing before. Its a separate way to make sure the org list is ok. 

also because after you revoke

Author: Yes, I see what you mean. This could be tried after each individual troubleshooting step. I'll make an experiment. :ok_hand:

Author: Please check the new version. :eyes:

# Why can't I see my organization?

> If you don't see your organization while [adding your organization on Codacy](../../organizations/what-are-synced-organizations.md#adding-an-organization), please try re-adding your Git provider or refreshing the list of organizations by clicking **Add provider** or **Refresh this list** on the Organizations page:

Reviewer 2: This wording feels repetitive: "organization while adding your organization", but not sure how to make it better...

Should this troubleshooting be moved to that page on "what-are-synced-organizations.md" instead?

Or maybe wording like:

"While [adding your organization on Codacy](https://deploy-preview-1051--docs-codacy.netlify.app/organizations/what-are-synced-organizations/#adding-an-organization) and it fails to appear in the page, please..."

But not sure about my suggestion either...

Author: Good point, here's a rewrite to make the text flow better:

```suggestion
If you can't [add your organization to Codacy](../../organizations/what-are-synced-organizations.md#adding-an-organization) because it doesn't appear on the Organizations page, please try re-adding your Git provider or refreshing the list of organizations by clicking **Add provider** or **Refresh this list** on the Organizations page:
```

What do you think @vascorsd?

Reviewer 2: I believe it flows much better with the added SEO you say :100:
